src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -1087,12 +1087,10 @@ public byte[] SerializeAdalV3()
         /// otherwise just use <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/>. 
         /// </summary>
         /// <param name="adalV3State">Array of bytes containing serialized Adal.NET V3 cache data</param>
-        /// <param name="merge">Future: Boolean enabling merging the cache into the existing cache.</param>
         /// <remarks>
         /// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
-        /// Merge = true is not yet available. This will be available in a later version of MSAL.NET.
         /// </remarks>
```

Comment:
></remarks> [](start = 12, length = 10)

For all deserialization methods, I suggest that there is a remark saying that the cache is merged into the existing cache ? Perhaps adding a few caveats to the comments?

Reply:
adalV3State: Array of bytes containing serialized Adal.NET V3 cache data to be merged into the in memory cache?

Reply:
adal v3 state isn't merged due to the nature of the cache.  i'm going to leave documentation here as is and we can discuss and do a follow up PR.

---
In reply to: [261829599](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/927#discussion_r261829599) [](ancestors = 261829599)